alfresco-transform-imagemagick/alfresco-transform-imagemagick-spring/pom.xml

```diff
@@ -1,14 +1,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>alfresco-docker-imagemagick</artifactId>
-    <name>Alfresco Docker ImageMagick</name>
+    <artifactId>alfresco-transform-imagemagick-spring</artifactId>
+    <name>Alfresco Transform ImageMagick Spring Boot</name>
     <packaging>jar</packaging>
 
     <parent>
         <groupId>org.alfresco</groupId>
         <artifactId>alfresco-transform-core</artifactId>
         <version>2.1.2-SNAPSHOT</version>
```

Comment:
We update the pom.xml (s) to 2.2.0-SNAPSHOT, eg.

mvn versions:set -DnewVersion=2.2.0-SNAPSHOT